pubmed_dl/main.py

```diff
@@ -1,20 +1,20 @@
-#!/usr/bin/env python
 import json
-
-from pubmed_dl.ncbi import get_list_pmid, uids_to_docs

```

Comment:
This shouldn't have been deleted. This is whats causing @jvwong 's error.

Reply:
Yes that fixed it thanks!

Reply:
Oh oops, I have matched the imports to the ```main.py``` file on the ```main``` branch. Should work now I think.

Reply:
However, this change is causing some merge checks to be unsuccessful. Is there something wrong??